packages/@o3r/rules-engine/builders/rules-engine-extractor/helpers/rules-engine.extractor.ts

```diff
@@ -277,6 +277,14 @@ export class RulesEngineExtractor {
               nbValues: 1
             }
           };
+          if (declaration.initializer && ts.isObjectLiteralExpression(declaration.initializer)) {
+            declaration.initializer.properties.forEach((property: any) => {
```

Comment:
`: any` should be avoid, how do you guaranty that `property.name` exists otherwise?